src/pudl_archiver/metadata/sources.py

```diff
@@ -33,7 +106,35 @@
         ),
         "working_partitions": {
             "years": [1991, 1994, 1998, 2002, 2006, 2010, 2014, 2018]
-        },  # Census DP1 is monolithic.
+        },
+        "keywords": sorted(
+            {
+                "manufacturing",
+                "MECS",
+            }
+        ),
+        "license_raw": LICENSES["us-govt"],
+        "license_pudl": LICENSES["cc-by-4.0"],
+        "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
+    },
+    "eiarecs": {
+        "title": "EIA RECS -- Residential Energy Consumption Survey",
+        "path": "https://www.eia.gov/consumption/residential/data/2020/",
+        "description": (
+            "EIA administers the Residential Energy Consumption Survey (RECS) to a"
+            "nationally representative sample of housing units. Traditionally,"
+            "specially trained interviewers collect energy characteristics on the"
+            "housing unit, usage patterns, and household demographics. For the 2020"
+            "survey cycle, EIA used Web and mail forms to collect detailed information"
+            "on household energy characteristics. This information is combined with"
+            "data from energy suppliers to these homes to estimate energy costs and"
+            "usage for heating, cooling, appliances and other end uses — information"
+            "critical to meeting future energy demand and improving efficiency and"
+            "building design. "
+        ),
+        "working_partitions": {
+            "years": [2009, 2015, 2020]  # Only PDFs: 1993, 1997, 2001, 2005
+        },
```

Comment:
Something went wrong here; lines 138-147 belong with MECS not RECS, but the MECS keywords are already updated above. RECS kws should be something like

```
            {
                "residential",
                "RECS",
            }
```

Reply:
ahh good catch! i copy/pasted and forgot to update. will update shortly!